__tests__/test_query01.js

```diff
@@ -0,0 +1,11 @@
+import { expect, describe, it, jest } from '@jest/globals';
+import './jest_extensions';
+
+jest.setTimeout(500000);
```

Comment:
(FYI, I usually use [Conventional Comment](https://conventionalcomments.org/) style for code comments)

**suggestion:** I think we should set a timeout limit of 60 seconds on each test by default [^1], _with the exception of query03, which should have a two-minute limit,_ as specified in the assignment readme.

If I'm not mistaken, the default could be set with a [`testTimeout`](https://jestjs.io/docs/configuration#testtimeout-number) configuration option in the _jest.config.mjs_ file.

[^1]: Just seems like a reasonable maximum -- long enough that the queries should be able to finish, but not so long that you'll wait an hour for a failure if you're on the wrong track. I would do without a timeout at all, but with these queries I know they can take a while to run, and if they're taking way too long, I'd rather let the student know that they should revisit their approach sooner than later.

Reply:
Sure, I'll reset the timeout